GetIntoTeachingApiTests/Services/StoreTests.cs

```diff
@@ -96,26 +104,25 @@ public async void SyncAsync_UpdatesExistingTeachingEvents()
         }
 
         [Fact]
-        public async void SyncAsync_DeletesOrphanedTeachingEventsAndBuildings()
+        public async void SyncAsync_DeletesOrphanedTeachingEvents()
         {
+            await SeedMockTeachingEventBuildingsAsync();

```

Comment:
Instead of doing this everywhere:

```
await SeedMockTeachingEventBuildingsAsync();
await SeedMockTeachingEventsAsync();
```

Can we just change `SeedMockTeachingEventsAsync` to `SeedMockTeachingEventsAndBuildingsAsync` and call the buildings seed from inside there to DRY it up a bit?